TODO.md

```diff
@@ -31,3 +31,17 @@
 - [x] Set up visibility such that all non-API symbols are hidden
 - [x] Rename
 - [ ] Look into cmake/clang-cl builds
+
+## R Package
+
+- [ ] Add hook for R package with formatR
+- [ ] Docs for all public functions
+- [ ] Vignette for usage
+- [ ] tinystan-tools companion package?
+- [ ] Add methods for running loo etc. with tinystan model
+- [ ] Verbose checking of inputs for all the algorithms
```

Comment:
Are there any inputs not being checked?

Reply:
They are all checked but this comment is to steal cmdstanr's checks which are very thorough and also return nice errors.